labs/b4.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A distribution is a combination of the a kernel, init system, libraries,
```

Comment:
Should probably explain (or at least link to the ArchWiki for kernel / init systems / library / drivers)